configs/components/module-puppetlabs-mount_core.json

```diff
@@ -1 +1 @@
-{"url":"git://github.com/puppetlabs/puppetlabs-mount_core.git","ref":"refs/tags/1.0.1"}
+{"url":"git://github.com/puppetlabs/puppetlabs-mount_core.git","ref":"refs/tags/1.0.2"}
```

Comment:
The commit message says it's bumping this one to 1.0.1, was that a typo or was this one not supposed to be bumped twice?

Reply:
The commit message was probably a copy/pasta error. Thanks for catching that!

Reply:
It's been updated!